Sources/TuistServer/OpenAPI/Client.swift

```diff
@@ -3669,6 +3669,123 @@ internal struct Client: APIProtocol {
             }
         )
     }
+    /// Change user's account name
+    ///
+    /// Updates the authenticated user's name
+    ///
+    /// - Remark: HTTP `PATCH /api/accounts/name`.
+    /// - Remark: Generated from `#/paths//api/accounts/name/patch(changeName)`.
```

Comment:
I'd have a single endpoint to update account instead of having endpoint per field.